protected/views/site/faq.php

```diff
@@ -315,256 +316,259 @@
                                 <div class="panel-body">
                                     <p>At the present time <em>GigaDB</em> doesn't have the resources to assist in data management plans, but there are many useful resources available on the internet, including places like the <a href="http://www.dcc.ac.uk/resources/data-management-plans">DCC (UK focus)</a>or <a href="https://www.cic.net/projects/technology/shared-storage-services/data-management-plans">the CIC (US focus)</a>.</p>
                                 </div>
                             </div>
                         </div>
                         <div class="panel panel-default">
                             <div class="panel-heading" role="tab">
-                                <h4 class="panel-title">
+                                <h2 class="h4 panel-title" id="heading26">
                                     <a role="button" data-toggle="collapse" data-parent="#accordion" href="#panel26" aria-expanded="false" aria-controls="panel26">
-                                    What are the charges for submitting data?    
+                                    What are the charges for submitting data?
                                     </a>
-                                </h4>
+                                </h2>
                             </div>
-                            <div id="panel26" class="panel-collapse collapse" role="tabpanel">
+                            <div id="panel26"  aria-labelledby="heading26" class="panel-collapse collapse" role="tabpanel">
                                 <div class="panel-body">
-                                    <p>There are currently no separate <u>d</u>ata <u>p</u>ublishing <u>c</u>harges (DPC's) for <em>GigaDB</em> as we currently do not accept data that is not accompanied by a <i>GigaScience </i>manuscript. All DPCs for <i>GigaScience </i>manuscripts are covered by the <u>a</u>rticle <u>p</u>ublishing <u>c</u>harges (APC's) of that manuscript (up to a terabyte automatically included, but contact us if you need more). For APCs of <i>GigaScience </i>manuscripts please see <a href="http://www.gigasciencejournal.com/about/apcfaq/howmuch">here</a>.</p>
+                                    <p>There are currently no separateData Publishing Charges (<abbr>DPCs</abbr>) for <em>GigaDB</em> as we currently do not accept data that is not accompanied by a <i>GigaScience </i>manuscript. All <abbr title="Data Publishing Charges">DPCs</abbr> for <i>GigaScience </i>manuscripts are covered by the Article Publishing Charges (<abbr>APCs</abbr>) of that manuscript (up to a terabyte automatically included, but contact us if you need more). For <abbr title="Article Publishing Charges">APCs</abbr> of <i>GigaScience </i>manuscripts please see in <a href="http://www.gigasciencejournal.com/about/apcfaq/howmuch">Gigascience journal pricing</a>.</p>
```

Comment:
the changes in this paragraph involve:

* wrapping abbreviatted terms in <abbr>
* removing the underlined text
* having a more explicit anchor text for the link